### PR TITLE
NEST-358: Disable Azure Media in development and testing

### PR DESCRIPTION
**Pull request by porgabi**
NEST-358

### CODE REVIEW

File: NuGetTest/src/Lombiq.OSOCE.NuGet.Web/appsettings.json [hunk 1]
```diff
@@ -7,14 +7,6 @@
...
   },
   "OrchardCore": {
-    "OrchardCore_DataProtection_Azure": {
-      "ConnectionString": "UseDevelopmentStorage=true",
-      "ContainerName": "osocedataprotection"
-    },
-    "OrchardCore_Media_Azure": {
-      "ConnectionString": "UseDevelopmentStorage=true",
-      "ContainerName": "osocemedia"
-    },
```
Comment (Piedone) [inline NuGetTest/src/Lombiq.OSOCE.NuGet.Web/appsettings.json]: Don't remove these. These are necessary for testing Azure-related features locally if we need to, but they don't bother if you don't already enable those features. Same for the root solution.

**Review by Piedone — CHANGES_REQUESTED** [review]

File: NuGetTest/src/Lombiq.OSOCE.NuGet.Web/appsettings.json (hunk 1 above)
Comment (Piedone) [reply NuGetTest/src/Lombiq.OSOCE.NuGet.Web/appsettings.json]: I'm rather reverting this.

Comment (porgabi) [reply NuGetTest/src/Lombiq.OSOCE.NuGet.Web/appsettings.json]: My bad, I was mistakenly certain these strings were added by me specifically for FeaturesGuard.

Comment (Piedone) [reply NuGetTest/src/Lombiq.OSOCE.NuGet.Web/appsettings.json]: You added them, but they're useful to keep (and in DotNest, they're necessary).

**Pull request merged**